### PR TITLE
add new rule win_process_dump_rdrleakdiag

### PR DESCRIPTION
**Pull request by kidrek**

### CODE REVIEW

File: rules/windows/process_creation/win_process_dump_rdrleakdiag.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+title: Process Dump via RdrLeakDiag.exe
+id: EDADB1E5-5919-4E4C-8462-A9E643B02C4B
```
Comment (frack113) [inline rules/windows/process_creation/win_process_dump_rdrleakdiag.yml:2]: Hi id is a uuidV4 lowercase

File: rules/windows/process_creation/win_process_dump_rdrleakdiag.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    - attack.credential_access
+    - attack.t1003.001
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        OriginalFileName: RdrLeakDiag.exe
+        CommandLine|contains:
+            - 'fullmemdmp'
```
Comment (frack113) [inline rules/windows/process_creation/win_process_dump_rdrleakdiag.yml:20]: As the list have only 1 value : `CommandLine|contains: fullmemdmp`
Get less brackets when convert

File: rules/windows/process_creation/win_process_dump_rdrleakdiag.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    condition: selection
+falsepositives: Unknown
```
Comment (frack113) [inline rules/windows/process_creation/win_process_dump_rdrleakdiag.yml:22]: falsepositives are [list](https://github.com/SigmaHQ/sigma/wiki/Specification#falsepositives)
```yaml
falsepositives: 
    - Unknown
```

**Review by frack113 — CHANGES_REQUESTED** [review]

File: rules/windows/process_creation/win_process_dump_rdrleakdiag.yml (hunk 1 above)
Comment (kidrek) [reply rules/windows/process_creation/win_process_dump_rdrleakdiag.yml:2]: Hi, ok, I obtained uuidV4 uppercase on my macOs ^^. The change has been made.

File: rules/windows/process_creation/win_process_dump_rdrleakdiag.yml (hunk 2 above)
Comment (kidrek) [reply rules/windows/process_creation/win_process_dump_rdrleakdiag.yml:20]: Ok, thx, the change has been made.

File: rules/windows/process_creation/win_process_dump_rdrleakdiag.yml (hunk 3 above)
Comment (kidrek) [reply rules/windows/process_creation/win_process_dump_rdrleakdiag.yml:22]: The change has been made.

**Review by frack113 — APPROVED** [review]

**Pull request merged**